src/test/java/io/numaproj/numaflow/function/reduce/ReduceSupervisorActorTest.java

```diff
@@ -57,29 +53,13 @@ public void invokeSingleActor() throws RuntimeException {
             supervisor.tell(inputDatum, ActorRef.noSender());
         }
 
-        Future<Object> resultFuture = Patterns
-                .ask(supervisor, Function.EOF, Integer.MAX_VALUE);
+        supervisor.tell(EOF, ActorRef.noSender());
 
-        List<Future<Object>> udfResults = null;
         try {
-            udfResults = (List<Future<Object>>)
-                    Await.result(resultFuture, Duration.Inf());
-        } catch (TimeoutException | InterruptedException e) {
-            throw new RuntimeException(e);
+            completableFuture.get();
+        } catch (InterruptedException | ExecutionException e) {
+            fail("Expected the future to complete without exception");
         }
-
-        assertEquals(udfResults.size(), 1);
-        udfResults.forEach(f -> {
-            try {
-                Message[] output = (Message[]) Await.result(f, Duration.Inf());
-                Arrays.stream(output).forEach(message -> {
-                    assertEquals(message.getKey(), Message.ALL);
-                    assertEquals(new String(message.getValue()), "10");
```

Comment:
Are we skipping output verification now?

Reply:
yes skipping it here, since I am testing only ReduceSupervisorActor. We verify the output in FunctionServerTest.